src/features/wallet/WalletModal/WalletModal.jsx

```diff
@@ -94,7 +94,7 @@ const WalletModal = ({ children, ...props }) => {
     }
 
     setModalTitle(t('walletModal.titleConnect'));
-  }, [walletView, error, account]);
+  }, [walletView, error, account, i18n.language]);
```

Comment:
why is it important for this modal? 🤔

Reply:
After changing  i18n.language with the switcher, we need to rerun this effect. We need it to change the modal title.
Otherwise, the modal title will remain in the locale that it was at the time of the first render.
This is because we use the t() function not in the template but inside the useEffect hook. My solution is working, but maybe it is not the best possible.

Reply:
thank you for explanation, that's okay. please also check if we have this problem somewhere else

Reply:
I have checked, this is the only such place, AFAIK.
I supposed that this will be the problem at the time of adding i18n. But I was not 100%, and it was hard to check without any locale switcher.